client/src/components/Search/SearchForm.js

```diff
@@ -26,11 +26,10 @@ function onEnter(callback) {
  * Display an advanced filter form.
  */
 function SearchForm({
-  visible, expanded, onSearch, onClear, formSchemaUrl, id, identifier, clearable
+  expanded, forceLoadForm, onSearch, onClear, formSchemaUrl, id, identifier, clearable
 }) {
   const handleKeyDown = onEnter(onSearch);
-  const loadForm = (visible || expanded);
```

Comment:
The "visible" state does not do what we need it to do. It's true by default which is what's causing all the unnecessary calls to the endpoint in the first place

It's not "visible" to the end user because it's hidden and toggled visible by css and jquery